WinUIGallery/SettingsPage.xaml.cs

```diff
@@ -80,7 +83,7 @@ private void OnSettingsPageLoaded(object sender, RoutedEventArgs e)
             NavigationRootPage navigationRootPage = NavigationRootPage.GetForElement(this);
```

Comment:
This declaration will cover the newly declared private field. Either use the field or change the variable name. Same for line 139 (use GetForElement(this) or cached variable?)